src/workloads/execution/GraphLookup.yml

```diff
@@ -1,18 +1,25 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  This test exercises the behavior of $graphLookup against a sharded foreign collection.
+  This test exercises the behavior of $graphLookup where either the local or foreign collection is
+  sharded. When both collections are unsharded, the pipeline should be moved to the
```

Comment:
Should we still have an UnshardedToUnsharded case in this workload? Same thing for the lookup sharded workload as well. It would be interesting to see the difference in this vs the unsharded environment, and useful to know for users who are using $lookup / $graphLookup with both collections unsharded but in a sharded cluster environment.

Reply:
Seems like the `*UnshardedEnvironment` test files are confusingly named, sorry -- they do run in an unsharded environment, but they also run in a sharded environment too (see the AutoRun param at the bottom of the files). So, we would not be increasing coverage if we added the UnshardedToUnsharded case to the GraphLookup and Lookup files.  I can rename those files in the next patch.

Does that address your point?

Reply:
Ahh, gotcha - sorry I missed that! In that case, I'd agree that the file names are slightly confusing haha. Could you rename them to `LookupWithOnlyUnshardedCollections` (or something less wordy) and also add a line about where each workload is running (sharded environment, replica set, etc) to the description of each benchmark?